tests/QMLEngine/qml/ScopeLeaf.qml

```diff
@@ -2,5 +2,5 @@ Item {
   property int value: 1
   property int rootValue: rootItem.value
   property int parentValue: parentItem.value
-  property int totalValue: rootValue + parentValue
+  property int totalValue: (rootValue + parentValue) * value
```

Comment:
Actually, this is from a commit that fixes the scopes test. It's does not belong to this PR, but it (the commit that fixes the scopes tests) is LGTM nevertheless.